src/test/java/io/vertx/core/file/FileSystemTest.java

```diff
@@ -1133,6 +1127,24 @@ public void testWriteAsync() {
     await();
   }
 
+  @Test(timeout = 5000)
```

Comment:
Did you check this timeout in Travis ? it sounds low to me and prone to false negatives

Reply:
The build passed https://travis-ci.org/github/eclipse-vertx/vert.x/builds/687019097

Actually I added this timeout so that it failed fast while the bug was not fixed on my box. I will remove it

Reply:
Done